README.md

```diff
@@ -1,8 +1,14 @@
+<div align="center">
+<h1> Blockchain Developer, Smart Contract, & Solidity Course - Powered By AI - Beginner to Expert Course | Foundry Edition 2023 </h1>
 
-# Blockchain Developer, Smart Contract, & Solidity Course - Powered By AI - Beginner to Expert Course | Foundry Edition 2023
+<a href="https://github.com/Cyfrin/foundry-full-course-f23"><img src="https://sloc.xyz/github/Cyfrin/foundry-full-course-f23" alt="LOC"/></a>
+<a href="https://github.com/Cyfrin/foundry-full-course-f23"><img src="https://visitor-badge.laobi.icu/badge?page_id=Cyfrin.foundry-full-course-f23" alt="visitor count"/></a>
```

Comment:
Could you remove this?

Reply:
Yeah, sure

Reply:
Do you want me to remove the visitors count and Lines of code badges right?